unitylibs/core/workflow/workflow-ai/widget.css

```diff
@@ -0,0 +1,420 @@
+.aside.text2image.unity-enabled {
```

Comment:
@hnvadobe I apologize if I have caused any confusion regarding `text2image` class. I spoke with @Ruchika4 on this and wanted to clarify that we should not introduce a new class (`text2image`) and rather leverage existing `unity-enabled` class from code for widget styles. As we want to generalize this workflow for more AI verbs in the future, we also should standardize the CSS accordingly without making it verb-specific. Please go ahead and remove `text2image` class.